src/Simulation/Core/Submitters/IQirSubmitter.cs

```diff
@@ -20,7 +20,9 @@ public interface IQirSubmitter : IAzureSubmitter
         /// <param name="qir">The QIR program as a byte stream.</param>
         /// <param name="entryPoint">The fully-qualified name of the entry point to execute.</param>
         /// <param name="arguments">The arguments to the entry point in the order in which they are declared.</param>
+        /// <param name="options">Additional options for the submission.</param>
         /// <returns>The submitted job.</returns>
-        Task<IQuantumMachineJob> SubmitAsync(Stream qir, string entryPoint, IReadOnlyList<Argument> arguments);
+        Task<IQuantumMachineJob> SubmitAsync(
```

Comment:
Similarly to [this](https://github.com/microsoft/qsharp-runtime/pull/673/files#r626707394) comment, I think this interface should also provide an option without the submission options.

Reply:
Closing given that now you provide a default instance for `SubmissionOptions`.